python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -101,6 +102,16 @@ def filterDefinitions(self):
         """
         return None
 
+    @property
+    @cached_getter
+    def checked_parameters(self):
+        # Docstring inherited.
+        parameters = super().checked_parameters
+        if "bbox" in parameters:
```

Comment:
Oh, interesting; that's a behavior possibility I hadn't considered.  It's tempting because it does sort of give the user what they asked for, but in this case, loading the whole image and then taking a subimage is so contrary to the _resource usage_ expectation of the user that it's probably safer to consider that an invalid implementation and just prevent code from trying it (I'm envisioning some huge processing job falling over because it used too much memory, and nobody being able to figure out why).  So I think the code as-is yields the least bad behavior we can provide (an exception), but I'm not sure that's a general statement about parameters.